client-side/C#/StreetSmartArcGISPro/StreetSmartArcGISPro/System/Linq/SystemCore_EnumerableDebugView.cs

```diff
@@ -0,0 +1,34 @@
+﻿/*
```

Comment:
You add a file, but this one is never used?

Reply:
I think I was trying to modify a method in there but didn't end up using it so I removed it and it seems to build fine.